src/fp2.rs

```diff
@@ -334,6 +334,33 @@ impl Fp2 {
         }
         res
     }
+
+    /// Attempts to convert a little-endian byte representation of
+    /// a scalar into an `Fp2`.
```

Comment:
I think this should be big endian to mirror the behavior of G2 point serialization. And it shouldn't mention a scalar.